axelrod/tests/unit/test_tournament.py

```diff
@@ -177,100 +159,18 @@ def test_parallel_play(self):
-            processes=4,
-            deterministic_cache=cache)
-        self.assertFalse(tournament._build_cache_required())
-
-        # Not noisy, empty deterministic cache
-        cache = axelrod.DeterministicCache()
-        tournament = axelrod.Tournament(
-            name=self.test_name,
-            players=self.players,
-            game=self.game,
-            processes=4,
-            deterministic_cache=cache)
-        self.assertTrue(tournament._build_cache_required())
-
-    def test_build_cache(self):
-        tournament = axelrod.Tournament(
-            name=self.test_name,
-            players=self.players,
-            game=self.game,
-            turns=200,
-            repetitions=self.test_repetitions,
-            processes=2)
-        tournament._run_single_repetition = MagicMock(
-            name='_run_single_repetition')
-        tournament._build_cache([])
-        tournament._run_single_repetition.assert_called_once_with([])
-        self.assertEqual(
-            tournament._parallel_repetitions, self.test_repetitions - 1)
-
-    def test_run_single_repetition(self):
-        interactions = []
-        tournament = axelrod.Tournament(
-            name=self.test_name,
-            players=self.players,
-            game=self.game,
-            turns=200,
-            repetitions=self.test_repetitions)
-        tournament._run_single_repetition(interactions)
-        self.assertEqual(len(tournament.interactions), 1)
-        self.assertEqual(len(tournament.interactions[0]), 15)
-
-    def test_run_serial_repetitions(self):
-        interactions = []
-        tournament = axelrod.Tournament(
-            name=self.test_name,
-            players=self.players,
-            game=self.game,
-            turns=200,
-            repetitions=self.test_repetitions)
-        tournament._run_serial_repetitions(interactions)
-        self.assertEqual(len(tournament.interactions), self.test_repetitions)
-
-    def test_run_parallel_repetitions(self):
-        interactions = []
-        tournament = axelrod.Tournament(
-            name=self.test_name,
-            players=self.players,
-            game=self.game,
-            turns=200,
-            repetitions=self.test_repetitions,
-            processes=2)
-        tournament._run_parallel_repetitions(interactions)
-        self.assertEqual(len(interactions), self.test_repetitions)
-        for r in interactions:
-            self.assertEqual(len(r.values()), 15)
+    #def test_run_parallel(self):
+        #tournament = axelrod.Tournament(
+            #name=self.test_name,
+            #players=self.players,
+            #game=self.game,
+            #turns=200,
+            #repetitions=self.test_repetitions,
+            #processes=2)
+        #tournament._run_parallel()
+        #self.assertEqual(len(tournament.interactions), 15)
+        #for r in tournament.interactions.values():
+            #self.assertEqual(len(r), self.test_repetitions)
```

Comment:
4b07252ed5672dc39d9066956d7ec07ddc902ee1 adds some tests for this. Basically I just check that the `True` is returned by the method itself but also using mocking that the `write_interactions` is called the amount of times we expect.